src/UI/Component/ViewControl/Factory.php

```diff
@@ -64,4 +64,38 @@ public function mode($labelled_actions, $aria_label);
 	 */
 	public function section(Button $previous_action, Button $button, Button $next_action);
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      The sortation view control enables users to change the order in which
+	 *      some data is presented.
+	 *      The common table-sorting mechanism, e.g., in which a column header is clicked,
+	 *      requires the column to be visible. Visibility and sorting can be decoupled
+	 *      by a control next to the actual table.
+	 *      This concept is used by many pages, notably in shops like e.g. amazon, and can
+	 *      be applied to not only tables but all kinds of data-listings.
+	 *   composition: >
+	 *      Sortation uses a Dropdown to display a collection of shy-buttons.
+	 *   effect: >
+	 *      A click on an option will change the ordering of the associated data-list
+	 *      by calling the current page with the parameter "sortation" and its
+	 *      value according to the selected option.
+	 *      The label displayed in the dropdown will be set to the selected sorting.
```

Comment:
If this is done async, the target container would itself need the ability to refresh it's own content, right?

Something like:
```
$container_to_be_sorted_according_to_a->withAsyncRenderUrl($url_with_a_as_sort_param);
$shy_button_a_in_dropdown->withOnClick(
    $container_to_be_sorted_according_to_a->getRefreshSignal();
);
```